dapp/src/features/liquidityPool/components/LiquidityPoolChart/LiquidityPoolChart.utils.ts

```diff
@@ -31,7 +31,7 @@ export function generateTraces({ data, filteredData, dateRange, priceRange }: Ge
   const customWidth = Math.round(barWidthPx * onePixel);
 
   const bars: Data & { base: any[] } = {
-    x: filteredData.map((x) => x.timestamp),
+    x: filteredData.map((x) => new Date(x.timestamp)),
```

Comment:
Maybe it would make sense to add some sort of util to handle dates, as in some places we use `new Date(...).toIsoString()` and in other its a simple `new Date()`?

Reply:
ISO date string is used everywhere except for a chart, as chart requires normal date to work with local time. Added a util for this. Having unit tests would probably help with clarifications.